src/request/components/request-detail-panel-logging.jsx

```diff
@@ -55,22 +80,22 @@ var LogMessages = React.createClass({
                 <thead>
                     <tr className="table-col-title-group">
                         <th width="5%"><span className="table-col-title">#</span></th>
-                        <th width="10%"><span className="table-col-title">Level</span></th>
+                        <th width="15%"><span className="table-col-title"><Icon fixedWidth="true" /> Level</span></th>
                         <th><span className="table-col-title">Message</span></th>
-                        <th width="10%"><span className="table-col-title">From Start</span></th>
-                        <th width="10%"><span className="table-col-title">Duration</span></th>
+                        <th width="15%"><span className="table-col-title">From Start (ms)</span></th>
+                        <th width="15%"><span className="table-col-title">Duration (ms)</span></th>
```

Comment:
I like it to, but it was removed. Hence why I wanted input from Chet as to if this was for a specific reason.